deployment/docker-build/pyaleph.dockerfile

```diff
@@ -75,13 +75,16 @@ COPY .git /opt/pyaleph/.git
 
 # Setup directories for `python setup.py develop`
 USER root
-RUN mkdir /opt/pyaleph/src/pyaleph.egg-info
+RUN mkdir -p /opt/pyaleph/src/pyaleph.egg-info
 RUN mkdir /opt/pyaleph/.eggs
 RUN chown -R source:source /opt/pyaleph/src /opt/pyaleph/.eggs /opt/pyaleph/.git
 
 # Install PyAleph source
 USER source
 WORKDIR /opt/pyaleph
+# TODO: replace by a proper install of p2pclient once the changes merged and released on the mainline repo
+RUN pip install -U --use-deprecated=legacy-resolver git+https://github.com/odesenfans/py-libp2p-daemon-bindings.git@c36b0262bf0b7581c0f9662c3f2fb4368e6b3c28
```

Comment:
Repository to migrate in `aleph-im`.

Reply:
There should be a release of the official p2pclient package soon, I am in contact with the maintainer.